tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -73,6 +80,20 @@ export class DataTableComponent implements OnDestroy {
     document.removeEventListener('dragover', preventDefault);
   }
 
+  ngAfterContentInit() {
+    this.syncHeaders();
+    this.headerCells.changes.subscribe(this.syncHeaders.bind(this));
+  }
+
+  syncHeaders() {
+    this.headerCells.forEach((headerCell) => {
+      headerCell.dragStart.subscribe(this.dragStart.bind(this));
```

Comment:
Can you ensure thee all get unsubscribed from when the component unmounts

Reply:
Turns out they were not. I added a structure to track them and ensure there are no leaks.